packages/tsconfig-reference/scripts/tsconfigRules.ts

```diff
@@ -168,15 +168,15 @@ export const relatedTo: [AnOption, AnOption[]][] = [
 
 function trueIf(name: string) {
   return [
-    `\`true\` if [\`${name}\`](#${name}) is \`true\``,
```

Comment:
Equally clear but less busy/wordy, I think?

Reply:
I think this is a reasonable call 👍🏻